traffic_monitor/towrap/towrap.go

```diff
@@ -365,8 +491,9 @@ func (s TrafficOpsSessionThreadsafe) trafficMonitorConfigMapRaw(cdn string) (*tc
 	return configMap, err
 }
 
-// LegacyTrafficMonitorConfigMap returns the Traffic Monitor config map from the Traffic Ops. This is safe for multiple goroutines.
-func (s TrafficOpsSessionThreadsafe) TrafficMonitorConfigMap(cdn string) (*tc.LegacyTrafficMonitorConfigMap, error) {
+// LegacyTrafficMonitorConfigMap returns the Traffic Monitor config map from the
```

Comment:
Yeah, you and Steve are pointing out a lot of those, and to address them once: that's generally because I didn't write it, in cases like this. I just wrapped it so I didn't need to scroll sideways to read it. But I can fix it.